packages/page-storage/src/Selection/Raw.tsx

```diff
@@ -27,30 +36,94 @@ function Raw ({ onAdd }: Props): React.ReactElement<Props> {
 
       setValue({
         isValid: u8a.length !== 0,
-        key: compactAddLength(u8a)
+        key,
+        keyAsBytes: compactAddLength(u8a)
       });
     },
     []
   );
 
+  const allStoragePrefixes = useMemo(
+    () => {
+      const palletNames = Object
+        .keys(api.query)
+        .map((value): string => (
+          value
+        ));
+      const allStoragePrefixesNotFlattened = palletNames.map((palletName) => {
```

Comment:
just a small note: you do not have to wrap single argument in parentheses (neither here nor in other places)

Reply:
Good point